### ISSUE TITLE
Error: Cannot find module '/tmp/tfjs-node'

### ISSUE THREAD

**Issue by lazovicff** [lucleray/tfjs-lambda#4]
Using `now dev` 15.7.0
MacOS 10.14.5
```
Unhandled rejection: { Error: Cannot find module '/tmp/tfjs-node'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at requireTf (/private/tmp/zeit-fun-36952c187fc17/score/index.js:10513:14)
    at createTfPromise (/private/tmp/zeit-fun-36952c187fc17/score/index.js:10527:12)
    at loadTf (/private/tmp/zeit-fun-36952c187fc17/score/index.js:10552:17)
    at module.exports.585.module.exports (/private/tmp/zeit-fun-36952c187fc17/score/index.js:14701:20)
    at calculateScores (/private/tmp/zeit-fun-36952c187fc17/score/index.js:19343:72)
    at searchSites (/private/tmp/zeit-fun-36952c187fc17/score/index.js:19391:26) code: 'MODULE_NOT_FOUND' }
LambdaError: RequestId: 226328a2-c467-45d9-be5e-da58e19029d4 Process exited before completing request
    at Lambda.<anonymous> (/snapshot/repo/dist/index.js:1:2535360)
    at Generator.next (<anonymous>)
    at fulfilled (/snapshot/repo/dist/index.js:1:2533501)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```
`AWS_LAMBDA_FUNCTION_NAME` exists with `now dev`, so it tries to create `tmp/tfjs-node` and it fails (permission error). Can we use another variable to check if its local or production env?

**Comment by lucleray** [comment]
We can check if the environment is `now dev` like this:
```js
const isNowDev = process.env.NOW_REGION === 'dev1'
```

I'll add this check to the library, thanks for filing the issue 🙂

**Comment by lucleray** [comment]
Published version `1.0.2` that should fix this issue.
